packages/preview/glossarium/0.5.7/README.md

```diff
@@ -0,0 +1,298 @@
+# Typst glossary
+
+> [!TIP]
+> Glossarium is based in great part of the work of [Sébastien d'Herbais de Thun](https://github.com/Dherse) from his master thesis available at: <https://github.com/Dherse/masterproef>.
+> His glossary is available under the MIT license [here](https://github.com/Dherse/masterproef/blob/main/elems/acronyms.typ).
```

Comment:
The file isn't available on the main branch anymore, maybe link to a specific git revision?